metrics/lib/common.bash

```diff
@@ -227,6 +227,17 @@ set_ksm_aggressive(){
 	sudo bash -c "echo ${KSM_AGGRESIVE_PAGES} > ${KSM_PAGES_FILE}"
 	sudo bash -c "echo ${KSM_AGGRESIVE_SLEEP} > ${KSM_SLEEP_FILE}"
 	sudo bash -c "echo 1 > ${KSM_ENABLE_FILE}"
+
+	# Disable virtio-fs and save whether it was enabled previously
+	set_virtio_out=$(sudo -E "${LIB_DIR}/../../.ci/set_kata_config.sh" shared_fs virtio-9p)
+	echo "${set_virtio_out}"
+	grep "already" <<< "${set_virtio_out}" > /dev/null || was_virtio_fs=true;
```

Comment:
Slightly shorter to use `grep -q` rather than redirection.

Reply:
Updated!